apps/pie-microsite/src/assets/styles/components/_tokensTable.scss

```
@@ -0,0 +1,121 @@
+@use '../fozzie-config' as f;
+
+$tokensTable-column-padding: f.spacing(d);
+$tokensTable-border-width: 1px;
+
+.c-tokensTable-list {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    width: 100%;
+
+    @include f.media('<=wide') {
+        margin-top: f.spacing(f);
+    }
```

I'd probably make your media queries consistent as to what is the default. I'd probably go mobile-first and put this in the base selector and then have:

```suggestion
    @include f.media('>wide') {
        margin: 0;
    }
```